### DISCUSSION TITLE
OSPOlogy Germany: OSPO Challenges and Trends (Day-1) - Summary Notes

### DISCUSSION THREAD

**Discussion by harshadsinkar** [todogroup/ospology#378] [OSPO Discussions]
**Introduction:** 

We recently had the [OSPOlogy Live event in Frankfurt](https://github.com/todogroup/ospology/tree/main/ospology-live/2023-october-germany), and I participated in roundtables about OSPO Challenges and Trends on Day-1.

Following discussion delved into ways to leverage open source beyond compliance and align it with our business objectives. We explore methods to support GenAI software and ethical usage, funding considerations, understanding the value of OSPO, and measurement strategies for success.

**Key Points:**

**1. Leveraging Open Source for Business Success**

- Share success stories to showcase open source's value.
- Invest in product-specific functionalities through open source.
- Determine when to invest more than just code contributions.
- Develop product-specific strategies.
- Utilize requirements management and identify key contacts.
- Compare open source efforts with competitors.
- Encourage development teams to contribute more to open source projects.

**2. Extending OSPO Support**

- Utilize influencers and champions to convey OSPO topics.
- Collaborate with competitors for mutual benefit.
- Consider industry-specific foundations like the Logistics Linux Foundation.
- Acknowledge that not all open source projects will align perfectly with every stakeholder.

**3. Ensuring Funding for OSPO**

- Allocate resources for regular OSPO activities.
- Formulate a budget based on calculated funding requirements.
- Occasionally use persuasive tactics for securing funding.

**4. Recognizing OSPO Value**

- One way to evaluate could be: OSPO's value is equivalent to the CTO's value.
- Promote knowledge sharing and involvement within the organization.
- Encourage overall involvement in open source initiatives.

**5. Measuring OSPO Success**

- Publish OSPO reports twice or once a year, including relevant metrics.
- Consider rewarding contributors based on the quality of their contributions.
- Strive for leadership in open source projects.
- Identify valuable open source projects and provide support.
- Share industry success stories to inspire in-house engagement.
- Stay attuned to industry trends to generate demand for open source initiatives.

**Conclusion:** 

The discussion highlights strategies to harness open source for business advantage, extend OSPO support, secure funding, understand its value, and measure OSPO success. These insights can help organizations optimize their approach to open source and achieve greater success.

**Comment by anajsana** [comment]
Thank you! ❤️ 

Added to the OSPOlogyLive knowledge achieve: https://github.com/todogroup/ospology/blob/main/ospology-live/2023-october-germany/README.md
